velox/vector/tests/DecodedVectorTest.cpp

```diff
@@ -40,6 +40,22 @@ class DecodedVectorTest : public testing::Test, public VectorTestBase {
     }
   }
 
+  void assertNoNulls(DecodedVector& decodedVector) {
+    ASSERT_TRUE(decodedVector.nulls() == nullptr);
+    for (auto i = 0; i < 100; ++i) {
+      ASSERT_FALSE(decodedVector.isNullAt(i));
+    }
+  }
+
+  void assertNulls(const VectorPtr& vector, DecodedVector& decodedVector) {
+    SCOPED_TRACE(vector->toString(true));
+    ASSERT_TRUE(decodedVector.nulls() != nullptr);
+    for (auto i = 0; i < 100; ++i) {
```

Comment:
nit maybe we can get size from the vector instead of hardcoding it here..

Reply:
Nice catch. Fixed.